src/app/ngrx/effects/cms.effects.ts

```diff
@@ -41,7 +43,36 @@ export class CmsEffects {
     })
   );
 
+  @Effect()
+  retryActions$: Observable<Action> = this.actions$.pipe(
+    ofType(ACTIONS.ActionTypes.CMS_ACCOUNT_RETRY_ACTIONS),
+    filter((action: ACTIONS.CmsAccountRetryActionsAction) => action.payload.actions && action.payload.actions.length > 0),
+    map((action: ACTIONS.CmsAccountRetryActionsAction) => action.payload),
+    mergeMap((payload: ACTIONS.CmsAccountRetryActionsPayload) => {
+      return this.auth.token.pipe(
+        first(),
+        mergeMap(token => {
+          if (token) {
+            if (!this.auth.parseToken(token)) {
+              return Observable.of(new ACTIONS.CmsAccountFailureAction({error: 'Permission denied'}));
+            } else {
+              payload.actions.forEach(action => this.store.dispatch(action));
+              return this.cms.individualAccount.map(doc => {
```

Comment:
Yeah, this is what i don't think you need.  The `prx-navuser` in the styleguide now gets its info [from the auth user-info service](https://github.com/PRX/styleguide.prx.org/blob/master/src/lib/src/header/navuser.component.ts#L2) (which is ID).  So shouldn't need to hit CMS at all.